openapi/endpoints/accounts/accounts-create.yaml

```diff
@@ -0,0 +1,26 @@
+tags:
+  - accounts
+summary: Create Account
+description: Create account
+security:
+    - $ref: '../../models/api-key-auth.yaml'
+requestBody:
+  content:
+    application/json:
+      schema:
+        $ref: './models/create-account-request.yaml'
+  required: true
+responses:
+  '200':
+    description: Account created successfully
+    content:
+      application/json:
+        schema:
+          $ref: './models/create-account-response.yaml'
+  '403':
```

Comment:
💬 Remark: It could be nice to have more detail on the possible error types here. For example: https://docs.immersve.com/api-reference/get-spending-prerequisites

It's also a bit strange that when you see an example from the schema for an error it is not a realistic example (this is an existing docs problem).
<img width="629" alt="image" src="https://github.com/immersve/immersve-docs/assets/64108933/0956bdbe-e3b7-4e61-9ced-1b03e5b46f76">

Reply:
Error messages aren't standardized in calcite (including this endpoint), they vary from some text like 'Error happened' to codes like 'ERROR_HAPPENED', would be good to implement a consistent approach there

Reply:
added good error messages